cartridge-cli.lua

```diff
@@ -3735,10 +3805,46 @@ function cmd_stop.callback(args)
         fiber.sleep(0.1)
     end
     if fio.stat(pid_file) then
-        assert(fio.unlink(pid_file))
+        if not fio.unlink(pid_file) then
+            warn('Failed to remove pid file (%s)', pid_file)
+        end
     end
     if fio.stat(args.console_sock) then
-        assert(fio.unlink(args.console_sock))
+        if not fio.unlink(args.console_sock) then
+            warn('Failed to remove console sock (%s)', args.console_sock)
+        end
+    end
+
+    return true
+end
+
+local function stop_all(args)
+    local instance_names = get_configured_isntances(args.cfg, args.app_name)
+    for _, instance_name in pairs(instance_names) do
+        local instance_args = table.copy(args)
+        instance_args.instance_name = instance_name
+        local ok, err = stop_one(instance_args)
+        if not ok then
+            return nil, string.format('Failed to stop %s: %s', instance_name, err)
```

Comment:
I'm not sure that we should do it a such way.

We should call "stop" for each instance and in case of errors we should accumulate error messages. E.g. we can't stop first instance, but we stop second and third and print error.
```
[instance_name_1] failed to stop: error_message1
[instance_name_4] failed to stop: error_message4
```